content/announcements/2025-04-22-kubecon-eu-wrapup.md

```diff
@@ -0,0 +1,60 @@
+---
+slug: cert-manager-kubecon-eu-2025-wrap-up
+title: Thank you KubeCon London
+description: A quick summary of cert-manager at KubeCon EU 2025!
+date: "2025-04-17T18:00:00Z"
+---
+
+KubeCon EU London 2025 is now behind us and we had a great time meeting the community.
+
+***Thank you to everyone who came by to meet the cert-manager team!***
+
+## Recap
+
+### How We Solved TLS at Scale: Self-Service, Multi-Tenant cert-manager
+
+You can now watch [Tim & Erik's talk](https://youtu.be/gWgagjHtnlE?si=mqUCMwMc04tF5Jlt) at your convenience on YouTube.
+
+For the last talk of the day at 17:30 London time, this was an extremely well attended talk!
+Even better were the questions and interest from the audience at the end.
+
+Congratulations Tim & Erik for a really well thought out and delivered talk.
+This talk may inspire your own PKI management enhancements.
+
+### Contribfest
+
+- **Attendees:** We had approximately 30 people join the cert-manager Contribfest.
+- **PRs:** 
+  - We had at least 6 PRs to various projects that afternoon 2nd April 2025. 
+  - We have had a great deal more contributions since such as [trust-manager #577](https://github.com/cert-manager/trust-manager/pull/577), [cert-manager #7678](https://github.com/cert-manager/cert-manager/pull/7678) and [cert-manager #7666](https://github.com/cert-manager/cert-manager/pull/7666).
```

Comment:
```suggestion
  - We have had a great deal more contributions since such as [trust-manager #577](https://github.com/cert-manager/trust-manager/pull/577), [trust-manager #582](https://github.com/cert-manager/trust-manager/pull/582) [cert-manager #7678](https://github.com/cert-manager/cert-manager/pull/7678) and [cert-manager #7666](https://github.com/cert-manager/cert-manager/pull/7666).
```